docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Agents for Amazon Bedrock
+description: Core utility
+---
+
+Author [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html#agents-how){target="_blank"} using event handlers and auto generation of OpenAPI schemas.
+
+<center>
+```mermaid
+--8<-- "docs/core/event_handler/bedrock_agents.mermaid"
```

Comment:
The diagram is good, but needs to be moved to somewhere with context.

Reply:
This follows the same patterns we use with other modern documentation pages, like [Data Masking](https://docs.powertools.aws.dev/lambda/python/latest/utilities/data_masking/)